NEWS.md

```diff
@@ -1,3 +1,6 @@
+## xxxx-xx-xx
+* [MODSOURMAN-384](https://issues.folio.org/browse/MODSOURMAN-384) Implement writing entities processing information to the log
```

Comment:
shouldn't it be included in the R1 release (v3.0.0)?

Reply:
Fixed.